### DISCUSSION TITLE
Governing Board Candidates

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre#78] [Governing Board Election]
The Governing Board elections will take place online on Thursday, August 25th, 2022 which coincides in time with FOSS4G. 

Candidates for the Governing Board can nominate themselves up to two weeks before the election, i.e., until Thursday, August 11th, 2022 by posting in this thread that they are interested in serving on the Board. Please include a description of your involvement in the MapLibre Project and disclose if you are paid for working on MapLibre or have a vested interested in the Project.

**discussion update** Each nomination should be a separate comment. Use the "reply" section under each nomination to ask questions.

**Comment by HarelM** [comment]
Hi everyone!
My name is Harel Mazor and I would like to nominate myself for the Governing Board.
I'm currently maintaining maplibre-gl-js (You might have seen some of my annoying comments if you have submitted a PR there :-)).
I'm working on an open source hobby project called Israel Hiking Map (co-founder) which is a small navigation app that uses Cordova and thus maplibre-gl-js to allow outdoor activities in Israel (hiking, biking, 4wd driving, running etc...).
I've been attending every steering committee meeting since the first time I joined after adding the `addProtocol` capabilities to maplibre-gl-js.
I've pushed forward the typescript migration along with the modernization of the maplibre-gl-js stack, and helped with the 3D terrain effort.
I generally enjoy participating in open source projects from the development side of things.
I'm currently employed at Dell as a group leader and previously done roles as a team leader and architect in the Semi-conductor industry.
Feel free to check out my linked-in profile here:
https://www.linkedin.com/in/harel-mazor-1298b139/
The project site here:
https://israelhiking.osm.org.il/
Or the project code here:
https://github.com/IsraelHikingMap/Site

Thanks! :-)

**Comment by lseelenbinder** [comment]
Hi all!

I'm Luke Seelenbinder, and it'd be my pleasure to have the opportunity to continue serving on the governing board.

As part of the founding group, I strongly believe that open-source map rendering libraries are valuable for the ecosystem (from individuals to large organizations), and that's why I initially led (along with my fearless fellow founding members) the push to ensure there was a single, vendor-neutral, and community-led fork of Mapbox GL JS, and I'm really excited to see the community and engagement grow over the last year+. So far, I've been most involved in the background, from Twitter-spamming at the beginning to build an initial group, to helping build the organizational charter, talking to sponsors, writing organizational docs and presentations, and ensuring we have a solid foundation for the future, etc.

If you choose to reelect me, I hope to achieve the following over the next term:

- Finalize the organization structure while supporting a governing board culture and habits that ensure it will succeed long-term
- See MapLibre GL Native become a much healthier project with near parity to GL JS
- Foster the continued development of MapLibre RS, a prototype of a future, unified cross-platform renderer

I use MapLibre projects daily via my [company](stadiamaps.com). Both personally and professionally, I see a lot of potential as the community continues to mature and collaborate.

Here's to the strong future of MapLibre! :rocket:

**Comment by nyurik** [comment]
I would like to nominate myself to the board of MapLibre. I feel it is important to show what I did so far, and explain how I see MapLibre's future. I have served on the board since MapLibre inception, so here it goes...
* I kick-started MapLibre via a Twitter [post](https://twitter.com/nyuriks/status/1336493514646052864). There are some good replies including the one from MapBox CEO just eight minutes after my initial tweet...
* Created MapLibre GitHub organization, set up OpenCollective banking and distributed donations acting as the organization treasurer
* Coordinated and worked on initial code rebranding, CI work, and initial NPM publishing, lots of work on [charter](https://github.com/maplibre/maplibre/pull/74/files) (thanks Oliver for drafting it!), ...
* Presented MapLibre at conferences like State of the Map US and the upcoming FOSS4G
* Engaged in numerous coordination and sponsorship talks with organizations like OSM board, Amazon, Meta (FB), Microsoft, and others, including those at the conferences. The recent large donation from Meta, and hopefully a few more upcoming ones are due in part to the SOTM-US conference hallway talks.
* Convinced several important repositories to join MapLibre:  MapLibre-RS, Martin, mbtileserver-rs, and hopefully soon -- Planetiler.
* As a developer, I mostly hack on the backend tooling like Martin - the tile server framework that should eventually serve from PG, mbtiles, and other formats, as well as support server side rendering.

Going forward, here is my vision and how I see my role...
* Listen to the community. Mediate between the needs of all contributors and supporters, large and small. Make sure we all align on the roadmap of the projects, both near and long term.
* Make sure MapLibre stays independent from any one company, and provides value to everyone who wants to do maps.
* I think MapLibre should supply the whole stack of features for developers, to go from the raw data like OSM and Natural Earth to the interactive map in a browser or apps. This means MapLibre should encourage innovation at all levels of the stack, provide tools to create and serve tiles, engage with new standards for 3D, projections, globe to flat zooms, experiment with joining native and javascript stacks (e.g. try Rust web assemblies), ... . The more interesting ideas we try, the more likely we will find the good ones bringing even more value (and fun) to everyone involved.
* No money for the board - I believe it would create too much conflict of interest if the board is allowed to decide on money allocation, and later turn around and receive that money. We all should decide for ourselves - do we want to represent the community and steer the projects forward, or do we want to receive payments for our efforts? I feel having both may put too much power in too few hands.

Your suggestions and feedback are welcome of course :)

**Comment by ianacl** [comment]
Hey everyone 👋 

My name is Ian Saultz, and I nominate myself to serve on the MapLibre Governing Board.

I work as a Software Engineer at AWS on a number of open source mobile SDKs and libraries. I was exposed to MapLibre last year while working on a geospatial related project. During that time, I made a few contributions to maplibre-gl-native and became a member of the technical steering committee shortly thereafter. I want to see MapLibre continue its success and help it reach the next level - I believe that the world needs a successful community-driven open source map rendering project. I’m not being paid to work on MapLibre (nor would I want to be) and I wouldn’t represent AWS as a Governing Board member. While I’m not immune to subconscious bias that may favor my employer, I’ll continue to do my best to keep it in check.

I’ve lived in the Apple ecosystem for the majority of my software development career, doing everything from iOS apps, to SDKs, to language/compiler work. In addition to the open source nature of my day job, I’m also a member of a Swift language workgroup working on Swift C++ interoperability.

Some personal info, so you know a little more about me. I live in Charleston, South Carolina, U.S.A with my wife and two children (expecting no. 3 any day now), and speak English and German.

My personal focus is on community building, improving processes around issues, PRs, & documentation, and getting MapLibre GL Native to the level we all want and know it can be. There are a lot of great ideas floating around about MapLibre GL Native; I hope to bring my experience and perspective to help the community evaluate these options and decide on the best path forward. As a member of the Governing Board, I see my foremost priorities as being a voice of the community and working towards longterm successes. 

Thanks!
Ian

**Comment by roblabs** [comment]
I am Rob Chohan and I would like to nominate myself as a candidate for the MapLibre Governing Board.  I am an independent Mobile Architect based in San Diego developing Open Source Geospatial solutions for Enterprise customers.  

I have been involved with MapLibre GL Native for Mobile since Winter 2020 when the project first kicked off.  My vision and approach to any Open Source code base is if the project a) has strong community support, b) is a joy for developers to use and create solutions with, c) and solves a problem for a customers.  A strong community means strong developer tools, which yields solid solutions for customers.

MapLibre and the original Mapbox project it was forked from is one of the key FOSS4G projects that meets this criteria.  I put MapLibre on the same page as QGIS, GDAL, and PostGIS for strategic Open Source projects for Geospatial.

I have been researching & developing products using Mapbox tools since 2016.  I have presented product research at FOSS4G World (2017), FOSS4G-NA (2018), Open Street Map US (2017 & 2018).  I am so inspired to aid the community that I was the Co-Chair and one of the driving forces for FOSS4G-NA in San Diego (2019).  To give back to the community I write researched technical blog posts, documentation & unit test commits, and code commits for optimal raster & vector data for offline map products.  In between Enterprise projects in 2021, I created [Maps for Non-Profits](https://RobLabs.com/blt) and a [Map Postcard generator](https://RobLabs.com/stw) for iMessage, also used by my Non-Profit friends.

My background is in image processing & optimization and product development.  In a previous industry, I was part of small team that created an Android print solution that has an install base of over 500 million devices.  I was proud to work at HP Labs and my GitHub handle is a hat tip to the great legacy of R&D from HP.

I make maps for web, mobile & print.  I want to get better maps in more peoples hands.

Thank you for allowing me to present my qualifications and for considering me for the Governing Board.  I look forward to many years of helping the MapLibre project.

Rob C. 

---


*The good product lifecycle:  needs Customers -> Developers -> Solutions*
[![](https://mermaid.ink/img/pako:eNpNj8FugzAMhl8F-VzKgI22OezS9gl6zcUlBqJBghIHqUK8-0LXsdkX__5-W_YMtVUEAjwj00Vj63BIp0KaJMY5eLYDOZ-k6WdyoYl6O0b5Q__0E99sH1hb86KbfMJtkzRrJv8CdhDBgFrFM-aVSeCOBpIgYqnQfUmQZom-MKp45VVptg5Eg72nHWBge3uYGgS7QL-m1yubq7eoKA7NwI9xfbjVnuPK2ppGt2s_uD62O-bRiyxb8b7V3IX7vrZD5rXq0HE3naqsKqojFiVVhxI_ylLV9_x0bIr3vFGHt7xAWJblG21xcKI)](https://mermaid.live/edit#pako:eNpNj8FugzAMhl8F-VzKgI22OezS9gl6zcUlBqJBghIHqUK8-0LXsdkX__5-W_YMtVUEAjwj00Vj63BIp0KaJMY5eLYDOZ-k6WdyoYl6O0b5Q__0E99sH1hb86KbfMJtkzRrJv8CdhDBgFrFM-aVSeCOBpIgYqnQfUmQZom-MKp45VVptg5Eg72nHWBge3uYGgS7QL-m1yubq7eoKA7NwI9xfbjVnuPK2ppGt2s_uD62O-bRiyxb8b7V3IX7vrZD5rXq0HE3naqsKqojFiVVhxI_ylLV9_x0bIr3vFGHt7xAWJblG21xcKI)

---

**Comment by birkskyum** [comment]
Hi, i'm Birk Skyum.

To support the organization and give you all some options, I'm nominating myself for the Governing Board.

I'm a co-founder of maplibre and I take part in the maintenance/bootstrapping of maplibre-gl-js and maplibre-gl-native. I have put in a ton of effort into modernizing our libraries over the past year, so that we are now ready to take them even further. I think MapLibre is uniquely positioned to support high quality visualizations in science, politics, tech and more, and this suite of open collaborative tools is in my opinion very important to have, and I care a lot that we can keep it open and expand upon the feature set to keep it relevant too.

My favorite things about our community is how friendly and welcoming it is, if we can keep our environment inviting, I know MapLibre as an umbrella will be a success. As a board member, I'll be focusing on getting the long-term strategical decisions for MapLibre right, such as a push towards webgpu when it's stabilizing, and on fostering collaboration and partnerships in our ecosystem. I'll also focus on the user story, because no matter if the map is supposed to run in with a specific web framework, flutter, expo, native or bevy, MapLibre should be trivial to include.

I typically use MapLibre the aid NGOs by building visual tools around public api's to democratise data. An example of such a project is this [agricultural map of Denmark](https://hvorerdyrene.dk/), which i.e. Greenpeace volunteers use as a key tool. I believe everybody who has a creative impulse should have the freedom to express it without a fee per map render.

Thank you for your time 🙏

**Comment by klokan** [comment]
Hi everybody.

Here is Petr Pridal alias Klokan. I would like to offer my services to the MapLibre project - as self-nominee for the governing board.
I would be honoured to continue to serve on the board - to assist in the further project growth and the next steps.

As one of the founders, I have had a chance to help to shape the MapLibre project into the current form.
My vision for the project is to make it the best rendering library for maps out there - used by millions of people every day - vendor/provider independent - and with true community governance.
Many of my ideas where to go practically are drafted in https://roadmap.maplibre.org/.
Some of them I have helped to make happen practically with the assistance of my team:

- First release of [MapLibre SDK for Android and iOS (Native)](https://www.maptiler.com/news/2021/06/maplibre-gl-native-open-source-mobile-sdk-for-android-and-ios/)
- Design project logo, visual identity, presentation template, first version of website https://maplibre.org/ with animated map and demotiles for CI and tests - all contributed to the project as there was a need for it
- Initial version of the pitch slidedeck & sponsorship presentation for successful fund-raising and setup of donations
- Promoting the project on conferences (SOTM, FOSS4G), in press (opensource.com,...) and on social media
- Technical steering committee organization and moderation, and active work on current board

I believe JavaScript and Native are the core and essential projects of the MapLibre organization, and especially Native needs more love and focus right now - to get to parity with JS. These are the cornerstones on which other projects grow - and the success of MapLibre depends on the success of the two. Focus is important - but there is a need for new MapLibre projects too (hopefully the upcoming Planetiler, continued innovation in Rust or other tools) which brings benefits to community and allow larger reach and will shape the project future, and they are also really cool technically.

I am an entrepreneur and software architect. I am a challenge seeker, Swiss resident born in the Czech Republic, and father of two boys. I hope to be able to contribute with my experience and from my position of CEO of MapTiler.com - acting on the board as a representative of the vendors who are using the MapLibre in production systems with thousands of customers. 
I have practical experience from board meetings, education in management and board governance from IMD in Switzerland, degree in Computer Science and PhD in cartography. Furthermore, my visionary mind and workaholism helped to shape OpenMapTiles.org, EPSG.io, GDAL2Tiles, OldMapsOnline.org, etc. More about me at https://www.linkedin.com/in/klokan/ and https://www.instagram.com/pridalpetr/.

Let's bring the MapLibre project to the next level - together :-)
